src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -117,6 +119,13 @@
      */
     @Override
     public boolean compile() throws IOException, InterruptedException {
+        String subscp = "";
```

Comment:
Can you rename to `substrateClasspath` or so?

Reply:
ok